utils/qemu/Makefile

```diff
@@ -96,6 +96,34 @@ define Package/qemu-bridge-helper/install
 	$(INSTALL_DATA) ./files/bridge.conf $(1)/etc/qemu
 endef
 
+define Package/qemu-img
+ SECTION:=utils
+ CATEGORY:=Utilities
+ SUBMENU:=Virtualization
+ TITLE:=QEMU Image utility
+ URL:=http://www.qemu.org
+ DEPENDS:=+glib2 $(CXX_DEPENDS) $(QEMU_DEPS_IN_HOST)
```

Comment:
Maybe target layerscape needs to be also added to `QEMU_DEPS_IN_HOST` and `QEMU_DEPS_IN_GUEST`.

By the way, can you share how you guys are using the platform for virtualisation?  I am very interested ;)

Reply:
@mcbridematt , good to know.  Expecting further news!